clang/lib/CConv/CConvInteractiveData.cpp

```diff
@@ -64,17 +80,64 @@ void ConstraintsInfo::print_stats(llvm::raw_ostream &O) {
       O << "{\"" << T.first << "\":{";
       O << "\"Num\":" << T.second.size() << ",";
       CVars TmpKeys;
-      findIntersection(InSrcWildPtrs, T.second, TmpKeys);
+      findIntersection(InSrcWildAtoms, T.second, TmpKeys);
       O << "\"InSrcNum\":" << TmpKeys.size() << ",";
       CVars InDWild, Tmp;
       InDWild = getWildAffectedCKeys(T.second);
-      findIntersection(InDWild, InSrcNonDirectWildPointers, Tmp);
+      findIntersection(InDWild, InSrcNonDirectWildAtoms, Tmp);
       O << "\"TotalIndirect\":" << InDWild.size() << ",";
-      O << "\"InSrcIndirect\":" << Tmp.size();
+      O << "\"InSrcIndirect\":" << Tmp.size() << ",";
+      O << "\"InSrcScore\":" << getAtomAffectedScore(Tmp);
       O << "}}";
       AddComma = true;
     }
     O << "]";
     O << "}";
     O << "}}";
-}
\ No newline at end of file
+}
+
+void ConstraintsInfo::printPerAtomStats(llvm::raw_ostream &O, Constraints &CS) {
```

Comment:
How does this function and the one for pointers compare to the vanilla `printStats` function? Is there code duplication that could be reduced?